scalableminds/webknossos
# Improve error message & more frontend analytics

This PR adds an additional error Toast for #5296 to better explain the error to the user and adds the requested analytics tracking of # 5445 except for "Store custom layout" as it does not make sense in my opinion as this is tightly coupled to "Customize layout (i.e. move panels, resize panels)" when the auto-saving layout is turned on (which is the default).

If you think tracking this event still makes sense just notify me :smile: 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Create a new project (`http://<>/projects/create`) 
- Then directly click on view in the project table.
- A tracing should open up with two errors:
    1. Annotation couldn’t be found
    1. It looks like this project does not have a single task completed. Make sure that at least one task of this project is finished to view it. 
I added the second error, to tell the user why this step failed and how to fix this error.

### Issues:
- fixes #5445, #5296

------
